### ISSUE TITLE
Delete from query documentation

### ISSUE THREAD

**Issue by eostis** [vespa-engine/vespa#26589]
Reading https://docs.vespa.ai/en/reference/document-v1-api-reference.html, I could not understand if deleting from query is possible.

My use case is to delete all documents with a specific document type.

**Comment by bratseth** [comment]
Yes, deleting with document/v1 is supported: https://docs.vespa.ai/en/reference/document-v1-api-reference.html#delete

**Comment by kkraune** [comment]
https://docs.vespa.ai/en/reference/document-v1-api-reference.html#delete-where - maybe the doc should be clearer, feedback welcome

**Comment by eostis** [comment]
Thanks. 

Indeed the documentation seems to lead to https://docs.vespa.ai/en/reference/document-v1-api-reference.html#selection, then to https://docs.vespa.ai/en/reference/document-select-language.html. But no examples.

Until I found in https://docs.vespa.ai/en/document-v1-api-guide.html:
curl http://localhost:8080/document/v1/namespace/music/docid?selection=music.genre=='blues'

I adapted it to my use case:

1. Delete all documents of a type
/document/v1/mynamespace/mytype/docid?cluster=mycluster&selection=true

12 Delete products or orders
/document/v1/mynamespace/mytype/docid?cluster=mycluster&selection=mytype.wpsolr_type = "product" or mytype.wpsolr_type = "shop_order"
